pulsar-broker/src/main/java/org/apache/pulsar/common/naming/NamespaceBundleFactory.java

```diff
@@ -377,6 +378,10 @@ public boolean canSplitBundle(NamespaceBundle bundle) {
         return range.upperEndpoint() - range.lowerEndpoint() > 1;
     }
 
+    public PulsarService getPulsar() {
```

Comment:
It's better not to make this method public, I see this method is only used in `NamespaceBundles`, we can use the default access modifier.

In addition, it would be more simple to create the strategy by the factory, e.g.

```java
    TopicBundleAssignmentStrategy createAssignmentStrategy() {
        return TopicBundleAssignmentFactory.create(pulsar);
    }
```